test/react.test.js

```diff
@@ -29,8 +29,7 @@ describe('styled', () => {
   })
 
   test('basic render with object as style', () => {
-    const fontSize = 20
-    const H1 = styled.h1({ fontSize })
+    const H1 = styled.h1({ fontSize: 20 })
```

Comment:
Wanted to call this change out: little hard to optimize object properties whose value is identifier as the value can be defined in same file or another. Maybe https://github.com/kentcdodds/css-in-js-precompiler would help down the road. Thoughts?